src/store/thunks/fetchArbeidsgivere.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  fetchArbeidsgivereError,
+  fetchArbeidsgivereStarted,
+  fetchArbeidsgivereSuccess
+} from "../actions/helseSpionActions";
+import { Dispatch } from "redux";
+
+export function fetchArbeidsgivere(): (dispatch: Dispatch) => Promise<void> {
+  return async dispatch => {
+    dispatch(fetchArbeidsgivereStarted());
+    await fetch('http://localhost:3000/api/v1/arbeidsgivere')
```

Comment:
Hardkodet url. Vil ikke virke i q-miljø dersom vi skal teste der.

Reply:
Yepp, mest for bare å teste fort.